### ISSUE TITLE
Demo site is closed

### ISSUE THREAD

**Issue by sblondon** [vifactor/repostat#207]
The README.md file links to https://repostat.imfast.io/ for a demo.
However, the demo has been replaced by a landing page from imfast.io saying:
```
Forbidden

The site you were looking for has been closed.
```

**Comment by vifactor** [comment]
Yeah, thanks I know, fast.io recently stopped support of free accounts. It
is possible to host the demo on github.io, but so far no time to prepare it.

чт, 7 січ. 2021, 15:31 користувач sblondon <notifications@github.com> пише:

**Comment by znerd** [comment]
Can we find the content somewhere else? If so, we can raise a GitHub issue, add the `goof-first-issue` label, and then perhaps someone else will be able to help.

**Comment by pulkomandy** [comment]
The content is just the output of running repostat on its own git repository. It is easy to regenerate
